apps/server/src/modules/board/uc/card.uc.spec.ts

```diff
@@ -82,7 +83,7 @@ describe(CardUc.name, () => {
 			it('should call the service to find the card', async () => {
```

Comment:
???

Reply:
For the error cases it would be 'when card service returns error' and 'when element service returns error'.

And to the existing case 'when creating a content element' the return value should be expected like:

```
const result = await uc.createElement(user.id, card.id, ContentElementType.TEXT);
expect(result).toEqual(expectedResult)
```

Reply:
@SevenWaysDP push nicht unserer Code, wir könnten hier aber die when Blöcke einfügen.